### PR TITLE
feat[disable-enable-account-#14]

### PR DESCRIPTION
**Pull request by FabCode67**
# task discription:
Due to varying reasons e.g fraud, abuse, malicious use or violation of terms of service, admin should be able to diable any user account.

GIVEN admin and multiple users exists on the sytem WHEN a user is suspected or proved to have violated the terms of service of the application THEN admin is able to trigger an action to block usage of a particular account and an email sent to the user with reasons fro the block.

# what PR do:

- logged in user can disable an account
- only signed in admin have permision to disable any user's acount
- only signed in admin has permision to inable disabled acount
- if user already enabled or disable will be notfied to an admin
- user will be disbled by assigning him status of inactive
- user will be enabled by assigning him status of active

# how to test it:
- git clone from branch
- npm run dev
- in your browser search localhost:{port}/docs

feat[finish disable-enable-account-#14]

### CODE REVIEW

File: src/controllers/Auth/user_status.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import  { user } from "../../models";
```
[the remark this answers is not in the document]
Comment (mukunzidd) [reply src/controllers/Auth/user_status.js:1]: @FabCode67 the` .hound.yml` is not referencing the aslant config correctly. It says `.eslintrc` while it's `.eslintrc.js`. Update that and below the config_file key add: 

`exclude: ["node_modules/**"]`

**Review by mukunzidd — CHANGES_REQUESTED** [review]
- Please read this https://github.com/atlp-rwanda/engineering-playbook/wiki/3.-Pull-Request and base on it to adjust your PR title. Look at the example and adjust to match.
- Look at the commit message formatting. Your commit should not look like the PR description.
- Fix hound issues by following my comment on the hound bot comment.

**Comment by FabCode67** [comment]
> * Please read this https://github.com/atlp-rwanda/engineering-playbook/wiki/3.-Pull-Request and base on it to adjust your PR title. Look at the example and adjust to match.
> * Look at the commit message formatting. Your commit should not look like the PR description.
> * Fix hound issues by following my comment on the hound bot comment.

I did it but when i try to rename the branch using command `git branch -m old_name new_name` and push it it creates the new branch with new name instead of updating the current branch , that is why i created other PR.

I Updated .hound.yml as you recommended me and I have used the resources to write standard pr and commit message.

**Comment by mansurissa** [comment]
@FabCode67  if this PR is a duplicate. Can you close it?

**Comment by FabCode67** [comment]
> @FabCode67 if this PR is a duplicate. Can you close it?

yea sure

**Pull request closed without merging**